CHANGELOG.md

```diff
@@ -1,27 +1,32 @@
 # Change Log
+
```

Comment:
Personally, I don't like this change, as it makes it harder to match headers to content.

In fact, I think it might be best to disable prettier for markdown.